src/components/radioButton/asRadioGroupChild.tsx

```diff
@@ -1,20 +1,9 @@
 import React, {Component} from 'react';
-import PropTypes from 'prop-types';
 import {RadioGroupContext} from './RadioGroup';
+import {RadioGroupChildPropTypes} from './types';
 
-export default function asRadioGroupChild(WrappedComponent) {
-  class RadioGroupChild extends Component {
-    static propTypes = {
-      /**
-       * The identifier value of the radio button. must be different than other RadioButtons in the same group
-       */
-      value: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
-      /**
-       * When using RadioButton without a RadioGroup, use this prop to toggle selection
-       */
-      selected: PropTypes.bool
-    };
-
+export default function asRadioGroupChild(WrappedComponent: React.ComponentType<any>) {
```

Comment:
I'm not sure this can be changed to something other than `React.ComponentType<any>` (and `any` below), it might be possible to make it `RadioButton` if we unify the files.

Reply:
that's fine, those internal typings are less important because they not what we expose to our users